.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I would still like all .idea/ files to not be included in the source code.  Each developer can create them if they desire.